packages/hash/api/src/model/knowledge/org.model.ts

```diff
@@ -1,4 +1,4 @@
-import { PropertyObject } from "@hashintel/hash-subgraph";
+import {EntityId, PropertyObject} from "@hashintel/hash-subgraph";
```

Comment:
Nit:
This doesn't look formatted, we may need to run prettier

Reply:
Oops, we certainly do, thanks

Reply:
Reran prettier on the repo, cheers